docs/RELEASE.md

```diff
@@ -0,0 +1,100 @@
+1. [ ]  release tag created (or maybe release branch, [discussion](https://github.com/WorldHealthOrganization/app/issues/279))
+1. [ ]  all automated tests pass
+1. [ ]  LQA tests
+1. [ ]  release notes generated
+
+\* Sometimes, engineering slightly changes the mockups for ease of implementation/scalability or runs into unforeseen issues on various devices, etc. — issues not generally caught by unit and integration tests. A release preview (whether URL or TestFlight release) should be provided to the design team with enough time to provide light feedback and discuss any implementation trade-offs. We imagine a few hours to be ideal for this, but more time is always welcome.
+
+## Compliance checks
+### Security checks ([discussion](https://github.com/WorldHealthOrganization/app/issues/269), [details](release/security_check_details.md))
+- [ ] System Boundary
+- [ ] APIs and Interconnection (external systems and services)
+- [ ] Infrastructure
+- [ ] Data Flow
+- [ ] Cryptography
+### Privacy checks ([discussion](https://github.com/WorldHealthOrganization/app/issues/280))
+Check how we stack up against these principles (qualitative evaluation, [details](release/privacy_check_details.md))
+- [ ] Collection Limitation Principle
+- [ ] Data Quality Principle
+- [ ] Purpose Specification Principle
+- [ ] Use Limitation Principle
+- [ ] Security Safeguards Principle
+- [ ] Openness Principle
+- [ ] Individual Participation Principle
+- [ ] Accountability Principle
+### Legal checks ([discussion](https://github.com/WorldHealthOrganization/app/issues/17))
+- [ ] 3rd party libraries / dependency license review
+- [ ] CLA / developer license review
+- [ ] Verify privacy policy vs actual system
+
+
+## Store material check ([details](release/store_asset_checks.md))
+
+1. [ ] iOS text assets committed
+1. [ ] android text assets committed
+1. [ ] iOS screenshots committed
+1. [ ] android screenshots committed 
+
+## Final WHO sign off/approval
+:construction: [waiting for input](https://github.com/WorldHealthOrganization/app/issues/274) from @brunobowden
+
+## Release processes
+
+### Prepare to release
+
+1. [ ] create release channel
+1. [ ] create release tag/branch ([discussion](https://github.com/WorldHealthOrganization/app/issues/279))
+1. [ ] :construction: create Google Drive to push release to WHO
+1. [ ] check build instructions for clarity and accuracy. :construction: wrap instructions in a single script
+1. [ ] determine the gatekeepers for the following categories (this may change as people join or leave)
+   1. [ ] engineering
+   1. [ ] design
+   1. [ ] compliance
+
+### Push to stores
+
+- [ ] get all [checkoffs](#checkoffs)
+- determine drop-dead time for:
+  - [ ] stopping code churn
+  - [ ] finalizing release in Github
+  - [ ] pushing build from GitHub to WHO
+- [ ] :construction: Upload all assets needed to Google Drive
+  - ‼️ Note that this should include all source code, not just binaries ‼️
```

Comment:
Why upload source code to Google Drive? The process so far has been that WHO has checked out the repo, then built and signed the binaries.

Reply:
Ah, the very first message from Bruno was that we would upload code as well (or at least that is how I interpreted it). you are right that they now check out the code directly. Fixed.